docs/building_the_project_with_dpcpp.rst

```diff
@@ -0,0 +1,470 @@
+   <https://github.com/oneapi-src/oneMKL?tab=readme-ov-file#software-requirements>`_. 
+
+.. _build_introduction_with_dpcpp:
+
+Build Commands
+###############
+
+In most cases, building oneMKL is as simple as setting the compiler and
+selecting the desired backends to build with.
+
+On Linux (see `Building for Windows`_ for building on Windows):
+
+.. code-block:: bash
+
+  # Inside <path to onemkl>
+  mkdir build && cd build
+  cmake .. -DCMAKE_CXX_COMPILER=$CXX_COMPILER                       \ # Should be icpx or clang++
+          -DCMAKE_C_COMPILER=$C_COMPILER                            \ # Should be icx or clang
+          -DENABLE_MKLGPU_BACKEND=False                             \ # Optional: The MKLCPU backend is True by default.
+          -DENABLE_MKLGPU_BACKEND=False                             \ # Optional: The MKLGPU backend is True by default.
+          -DENABLE_<BACKEND_NAME>_BACKEND=True                      \ # Enable any other backend(s) (optional)
+          -DENABLE_<BACKEND_NAME_2>_BACKEND=True                    \ # Multiple backends can be enabled at once.
+          -DBUILD_FUNCTIONAL_TESTS=False                            \ # See page *Building and Running Tests* for more on building tests. ON by default.
+          -DBUILD_EXAMPLES=False                                      # Optional: On by default.
+  cmake --build .
+  cmake --install . --prefix <path_to_install_dir>                    # required to have full package structure
+
+In the above, the ``$CXX_COMPILER`` and ``$C_COMPILER`` should be set to
+``icpx`` and ``icx`` respectively when using the Intel(R) oneAPI DPC++ Compiler,
+or ``clang++`` and ``clang`` respectively when using the Open DPC++ Compiler. 
+
+Backends should be enabled by setting ``-DENABLE_<BACKEND_NAME>_BACKEND=True`` for
+each desired backend. By default, only the ``MKLGPU`` and ``MKLCPU`` backends
+are enabled. Multiple backends for multiple device vendors can be enabled at
+once (albeit with limitations when using portBLAS and portFFT). For examples,
+see the section `CMake invocation examples`_. The supported backends for the
+compilers are given in the table at `oneMKL supported configurations table
+<https://github.com/oneapi-src/oneMKL?tab=readme-ov-file#supported-configurations>`_,
+and the CMake option names are given in the table below. Some backends may
+require additional parameters to be set. See the relevant section below for
+additional guidance.
+
+If a backend library supports multiple domains (i.e. BLAS, LAPACK, DFT, RNG), it
+may be desirable to only enable selected domains. For this, the
+``TARGET_DOMAINS`` variable should be set. See the section `TARGET_DOMAINS`_.
+
+By default, the library also additionally builds examples and tests. These can
+be disabled by setting the parameters ``BUILD_FUNCTIONAL_TESTS`` and
+``BUILD_EXAMPLES`` to off. Building the functional tests may require additional
+external libraries. See the section :ref:`building_and_running_tests` for more
```

Comment:
```suggestion
be disabled by setting the parameters ``BUILD_FUNCTIONAL_TESTS`` and
``BUILD_EXAMPLES`` to ``False``. Building the functional tests requires additional
external libraries for BLAS and LAPACK domains. See the section :ref:`building_and_running_tests` for more
```

Reply:
The false was fixed in [8f4fc93](https://github.com/oneapi-src/oneMKL/pull/510/commits/8f4fc934065a380cae8462b00998e8f08866ba06).

The other change was made in [ef02116](https://github.com/oneapi-src/oneMKL/pull/510/commits/ef021162bfa1dd44b8746c8b214cf248b16cbdf9).